en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml

```diff
@@ -3444,226 +3554,173 @@ paths:
       parameters:
         - $ref: '#/components/parameters/apiId'
-        - $ref: '#/components/parameters/mediationPolicyId'
-        - $ref: '#/components/parameters/If-None-Match'
       responses:
         200:
           description: |
             OK.
-            Mediation policy returned.
+            Requested complexity details returned.
           headers:
-            ETag:
-              description: |
-                Entity Tag of the response resource.
-                Used by caches, or in conditional requests (Will be supported in future).
-              schema:
-                type: string
-            Last-Modified:
-              description: |
-                Date and time the resource has been modifed the last time.
-                Used by caches, or in conditional requests (Will be supported in future).
-              schema:
-                type: string
             Content-Type:
               description: |
-                The content type of the body.
+                The content of the body.
               schema:
                 type: string
+                default: application/json
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Mediation'
-        304:
-          description: |
-            Not Modified.
-            Empty body because the client has already the latest version of the requested resource (Will be supported in future).
-          content: {}
+                $ref: '#/components/schemas/GraphQLQueryComplexityInfo'
         404:
           $ref: '#/components/responses/NotFound'
-        406:
-          $ref: '#/components/responses/NotAcceptable'
       security:
         - OAuth2Security:
             - apim:api_view
-            - apim:mediation_policy_view
+            - apim:api_create
+            - apim:api_manage
+            - apim:api_publish
       x-code-samples:
         - lang: Curl
           source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/publisher/v2/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/mediation-policies/f56eb8b4-128c-45aa-ad35-9c87a546261a"'
+          "https://127.0.0.1:9443/api/am/publisher/v3/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/graphql-policies/complexity"'
+      operationId: getGraphQLPolicyComplexityOfAPI
 
-    delete:
+
+    put:
       tags:
-        - API Mediation Policy
-      summary: Delete an API Specific Mediation Policy
+        - GraphQL Policies
+      summary: Update Complexity Related Details of an API
       description: |
-        This operation can be used to delete an existing API specific mediation policy providing the Id of the API and the Id of the mediation policy.
-      operationId: deleteAPIMediationPolicyByPolicyId
+        This operation can be used to update complexity details belonging to an API by providing the id of the API.
```

Comment:
- "by providing the API ID." sounds better

Reply:
Fixed.